.gitignore

```diff
@@ -6,6 +6,7 @@ Session.vim
 .netrwhist
 *~
 /debs/
+.junk/
```

Comment:
Take this out.

Reply:
fair enough
i was in a bit of a hurry...